### ISSUE TITLE
Issue with debugging pinn: Recovery shell doesn't supports other keyboards

### ISSUE THREAD

**Issue by ghost** [procount/pinn#321]
If you press ctrl-alt-f2 on an running PINN system and try to login and have a QWERTZ keybord, it uses QWERTY layout even if you chose QWERTZ in PINN.

**Comment by ghost** [comment]
I hopefully can fix this with new buildroot, I change some things in the config.

**Comment by lurch** [comment]
Interesting. I suspect the recovery shell is using the keyboard layout chosen by the Linux kernel / console / bash / busybox setup, and changing the keyboard in "PINN" changes the keymap at the Qt (GUI) layer, but doesn't change any of the lower layers.

**Comment by ghost** [comment]
Yes, it could be fixable by removing the "purge unwanted locales" option in buildroot.

**Comment by ghost** [comment]
If it's not fixable, it would possible to add a warining to README or change the default password
